components/self-service/new-content-done.tsx

```diff
@@ -80,6 +80,13 @@ export default function NewContentDone() {
         qe_owner: inputs.deliveryRepoQeOwner,
         release_category: inputs.deliveryRepoReleaseCategory,
         summary: inputs.deliveryRepoSummary,
+        application_categories: inputs.deliveryRepoApplicationCategories,
+        usage_type: inputs.deliveryRepoImageUsageType,
+        multistream: inputs.deliveryRepoContentStructure === 'multistream',
+        // List content_stream_tags only if multistream is chosen
```

Comment:
when Content Structure chose multistream this is correct
```
content_stream_tags:
    - v4.15
    - v4.16
```
but when chose single stream, content_stream_tags also needed, it should to be
```
content_stream_tags:
  - latest
```

Reply:
Totally missed that, thank you! Should be fixed now